doc/source/contributing.rst

```diff
@@ -150,16 +150,16 @@ after updating.
 
 2) Creating a development environment
 ---------------------------------------
-A development environment is a virtual space where you can keep an independent installation of *geopandas*.
+A development environment is a virtual space where you can keep an independent installation of *GeoPandas*.
 This makes it easy to keep both a stable version of python in one place you use for work, and a development
 version (which you may break while playing with code) in another.
 
-An easy way to create a *geopandas* development environment is as follows:
+An easy way to create a *GeoPandas* development environment is as follows:
 
 - Install either `Anaconda <http://docs.continuum.io/anaconda/>`_ or
   `miniconda <http://conda.pydata.org/miniconda.html>`_
 - Make sure that you have :ref:`cloned the repository <contributing.forking>`
-- ``cd`` to the *geopandas** source directory
+- ``cd`` to the *GeoPandas** source directory
```

Comment:
this should actually better stay lower case I think, as the directory name is lower case

Reply:
:) funny i just was looking at that thinking the same. you are totally right. it's confusing otherwise!!